homeassistant/components/home_connect/coordinator.py

```diff
@@ -626,39 +626,34 @@ def refreshed_too_often_recently(self, appliance_ha_id: str) -> bool:
         """Check if the appliance data hasn't been refreshed too often recently."""
 
         now = self.hass.loop.time()
-        if len(self._execution_tracker[appliance_ha_id]) >= MAX_EXECUTIONS:
-            return True
+
+        execution_tracker = self._execution_tracker[appliance_ha_id]
+        initial_len = len(execution_tracker)
 
         execution_tracker = self._execution_tracker[appliance_ha_id] = [
             timestamp
-            for timestamp in self._execution_tracker[appliance_ha_id]
+            for timestamp in execution_tracker
             if now - timestamp < MAX_EXECUTIONS_TIME_WINDOW
         ]
 
         execution_tracker.append(now)
 
         if len(execution_tracker) >= MAX_EXECUTIONS:
-            ir.async_create_issue(
-                self.hass,
-                DOMAIN,
-                f"home_connect_too_many_connected_paired_events_{appliance_ha_id}",
-                is_fixable=True,
-                is_persistent=True,
-                severity=ir.IssueSeverity.ERROR,
-                translation_key="home_connect_too_many_connected_paired_events",
-                data={
-                    "entry_id": self.config_entry.entry_id,
-                    "appliance_ha_id": appliance_ha_id,
-                },
-                translation_placeholders={
-                    "appliance_name": self.data[appliance_ha_id].info.name,
-                    "times": str(MAX_EXECUTIONS),
-                    "time_window": str(MAX_EXECUTIONS_TIME_WINDOW // 60),
-                    "home_connect_resource_url": "https://www.home-connect.com/global/help-support/error-codes#/Togglebox=15362315-13320636-1/",
-                    "home_assistant_core_issue_url": "https://github.com/home-assistant/core/issues/147299",
-                },
-            )
+            if initial_len < MAX_EXECUTIONS:
```

Comment:
Why is the check "less than" max executions?

Reply:
It 's to avoid the warning log being logged multiple times without being enabled again (once for each time that the updates are disabled)

Reply:
Ok.